### PR TITLE
Add Merchant Serial Number to the headers

### PR DESCRIPTION
**Pull request by rebeksburnett**
The file version was also updated, so description fields were added automatically.

### CODE REVIEW

File: tools/vipps-login-api.postman_environment.json [hunk 1]
```diff
@@ -1,47 +1,40 @@
...
 			"key": "scope",
 			"value": "openid",
-			"description": "Requested scopes",
 			"enabled": true
 		},
 		{
 			"key": "code",
 			"value": "\"Add code from url\"",
-			"description": "Code used to fetch access token after finishing a Vipps login",
 			"enabled": true
```
Comment (niklud) [inline tools/vipps-login-api.postman_environment.json:38]: Why remove description?

File: tools/vipps-login-api.postman_collection.json [hunk 2]
```diff
@@ -40,6 +40,11 @@
 			"request": {
 				"method": "GET",
 				"header": [
+					{
+						"key": "Merchant-Serial-Number",
+						"value": "{{merchantSerialNumber}}",
+						"type": "default"
+					},
```
Comment (niklud) [inline tools/vipps-login-api.postman_collection.json:47]: Have you tested that it works sending these? Or at least that nothing breaks?

**Review by niklud — APPROVED** [review]
Looks fine. I left some comments, but ignore them if they are irrelevant.

File: tools/vipps-login-api.postman_environment.json (hunk 1 above)
Comment (rebeksburnett) [reply tools/vipps-login-api.postman_environment.json:38]: I didn't notice that it removed those! I didn't do it  myself.

Comment (rebeksburnett) [reply tools/vipps-login-api.postman_environment.json:38]: I have added them back manually. Maybe the field is no longer present in a newer version of postman.

File: tools/vipps-login-api.postman_collection.json (hunk 2 above)
Comment (rebeksburnett) [reply tools/vipps-login-api.postman_collection.json:47]: yes

File: tools/vipps-login-api.postman_environment.json (hunk 1 above)
Comment (rebeksburnett) [reply tools/vipps-login-api.postman_environment.json]: I added them back manually.

**Pull request merged**